htdocs/core/lib/xcal.lib.php

```diff
@@ -404,6 +407,10 @@ function build_rssfile($format, $title, $desc, $events_array, $outputfile, $filt
 				fwrite($fichier, "<category><![CDATA[".$category."]]></category>\n");
 				fwrite($fichier, "<description><![CDATA[");
 
+				if (!empty($image)) {
+					fwrite($fichier, '<p><img class="center" src="'.$image.'"/></p>');
```

Comment:
When parameter is coming from $event_array['image'], we can say it is a public image url.
But when parameters even_array is list of WebsitePage, the property "image" is just a relative path. For example "image/template-corporate/background_dolibarr.jpg"
So the link will not works.  You should forge the url of image with 
$website->virtualhost.'/medias/'$image;   

$website is a website object parent of $websitepage.
$website->virtualhost contains for example:  https://www.dolicloud.com